gradle.properties

```diff
@@ -15,4 +15,5 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+android.useAndroidX=true
```

Comment:
Make sure to enable this setting to add a line break at the end of each file
![Screen Shot 2022-06-30 at 10 50 08 am](https://user-images.githubusercontent.com/13775137/176569579-0ee8f25c-873a-4691-821d-82605c8f8e38.png)

Reply:
Thanks i did the changes in studio settings